mne_bids/tests/test_read.py

```diff
@@ -692,8 +693,6 @@ def test_get_matched_emptyroom_no_meas_date():
     write_raw_bids(raw, bids_basename, bids_root, overwrite=True)
     os.remove(op.join(bids_root, 'participants.tsv'))
 
-    from mne_bids.path import print_dir_tree
-    print_dir_tree(bids_root)
     with pytest.warns(RuntimeWarning, match='Could not retrieve .* date'):
```

Comment:
@hoechenberger without turning off the check for `subject='emptyroom'`, this test fails. I'm a bit uncertain how to best tackle that. 

The bids_basename here has an invalid session entity, and it seems that the emptyroom is trying to retrieve the date to set the session, but in read_raw_bids, a call to `update()` will error out before it can do this.

Reply:
nvm. there was this one line in `get_matched_empty_room` that could be removed and fixed everything!